packages/voyager-audit/package.json

```diff
@@ -21,7 +21,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@aerogear/voyager-tools": "^0.4.0",
+    "@aerogear/voyager-tools": "^0.5.0-rc1",
```

Comment:
This is going to trigger renovate as it will expect all things pinned.

Reply:
Strange that it hasn't done that before 🤔